nnvm/python/nnvm/frontend/onnx.py

```diff
@@ -477,7 +518,7 @@ def _get_convert_map(opset):
         'SpatialBN': BatchNorm.get_converter(opset),
 
         # defs/generator
-        # 'Constant'
+        # 'Constant' # Implemented
```

Comment:
What is this change?

Reply:
Already implemented but it's not part of conversion map. Hence marked as Implemented.